src/routes/message.route.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+import { AuthMiddleware } from '@/middlewares';
+import { Routes } from '@/interfaces';
+import { MessageController } from '@/controllers';
+
+class MessageRoute implements Routes {
+  public readonly path = '/messages';
+  public readonly router = Router();
+  private messageController = new MessageController();
+
+  constructor() {
+    this.initializeRoutes();
+  }
+
+  private initializeRoutes() {
+    this.router.get(`/chatroom/:chat_id`, AuthMiddleware, this.messageController.getChatroomMessages);
```

Comment:
I think this should be /chatroom/:chat_id/this.path

Reply:
Makes sense